ddtrace/llmobs/_llmobs.py

```diff
@@ -736,7 +736,9 @@ def export_span(cls, span: Optional[Span] = None) -> Optional[ExportedLLMObsSpan
                 error = "invalid_span"
                 log.warning("Span must be an LLMObs-generated span.")
                 return None
-            return ExportedLLMObsSpan(span_id=str(span.span_id), trace_id=format_trace_id(span.trace_id))
+            return ExportedLLMObsSpan(
+                span_id=str(span.span_id), trace_id=format_trace_id(span._get_ctx_item(LLMOBS_TRACE_ID))
```

Comment:
should we `or span.trace_id`? this gets us a bit back into that realm of what should we fall back to...

Reply:
not adding a test case for now bc i'm not sure how we'd end up in that situation